### DISCUSSION TITLE
SLACKES SCAMMER<title>

### DISCUSSION THREAD

**Discussion by Slackescammer** [community/community#45533] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

Hello. Sorry for my english in first. I must tell you what happened. I Google spoofer on browser andi found a profile here on GitHub. The profile name is SLACKES. This a**hole claim to be a very good programmer. I contact him on discord. Initially he was very polite and professional. But After i gave him Money he disappear and i ban me and block me everywhere. I know scammers are everywhere and maybe i deserve the scam but  guys i must inform everyone. SLACKES IS A SCAMMER

**Comment by samuel-ma** [comment]
Thanks for reaching out to the community @Slackescammer 
I see you've even personally created an account to report the above user ###SLAKES
Please try reporting this issue to a stuff member in more details
You can reach out on this community forum

**Reply by Slackescammer** [reply]
Where i can report him?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Slackescammer 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms.  I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
